test/utils.dart

```diff
@@ -9,6 +9,7 @@ import 'package:test/test.dart';
 /// with the given [message].
 Matcher throwsStringScannerException(String text) {
   return throwsA(predicate((error) {
+    // ignore: deprecated_member_use
```

Comment:
Why not fix this? @chalin ?

Reply:
Because this package spans Dart 1 and Dart 2 so we can't use the new API (`TypeMatcher`).

Reply:
ack – thanks for clarifying!